use a confidir instand of a file

satan1st opened the pull request.

Commit:
use a confidir instand of a file

cbdevnet commented:
Doku anpassen waer noch super, am besten waer vermutlich einfach den User waehlen zu lassen ob er Conffile oder Confdir will...

Problem bei Confdir ist halt einmal dass remotes die mit '/' anfangen (geklont mit ssh://foo.bar/baz) nicht von welchen ohne (geklont mit ssh://foo.bar:baz/) unterschieden werden koennen und dass man prinzipiell 3 Files pro Repo braucht (Config, Read-Users, Write-Users)

cbdevnet commented on fugit:
```diff
@@ -1,25 +1,19 @@
 push(){
 	#read repo config
-	eval $(grep -F "REPO $1$" $CONFFILE -A 3 | tail -n +2)
+	eval $CONFDIR/$REPO.conf
```
Hier sollte glaube statt eval das conffile sourced werden

Commit:
remove all path information from REPO

Now all repo urls are possible, but you can't have two repos
where basename $REPO1 == basename $REPO2

Commit:
Merge branch 'master' into confdir

ghost commented:
Das Problem mit dem '/' ist gelöst per basename gelöst.
Doku pass ich morgen an.
Ps: Es reichen 2 Files pro repo (config, Users), dann darf halt jeder lesen und schreiben

Commit:
changed the Docs and examples for confdir

Commit:
use path information to identify repo

Now you can have multible repos with the same name
in diffrent folders. For that the the path delimiter is
replaced by a '-'.

ghost commented:
Jetzt wird der ganze Pfad für den Namen des config Files benutzt.
Dass man 3 (2) Files pro Repo braucht/hat ist grad der Punkt warum ich diese änderung gemacht
habe, da ich die userverwaltung irgendwann per mail halbautomatisieren möchte. Da ist es leichter wenn ich für die Nutzer extra Files habe.

Commit:
add check_acl to choos between conffile and dir

Which mechanism is used is choosen by the CONF variable.
